.github/workflows/docker.yml

```diff
@@ -32,13 +32,13 @@ env:
 jobs:
 
   # Only run for push to configured branches, do not run for releases.
-  # Take care of different os. For master branch, created tags are:
+  # Take care of different os. For main branch, created tags are:
   # latest-alpine, latest-debian, latest-ubuntu
   # For releasebranch_7_8, created tags are:
   # stable-alpine, stable-debian, stable-ubuntu
   docker-branch-os-matrix:
     name: build and push ${{ matrix.os }} for branch
-    if: startsWith(github.ref, 'refs/tags/') != true
+    if: startsWith(github.ref, 'refs/heads/') != true
```

Comment:
This job is meant to run for everything except tags. Now it prevents the run for branches which is needed here. I would agree if the negotioation is removed (`!= true`) then maybe it is more clear. The check for `refs/heads/` should be sufficient, I don't think any other ref type is relevant here.

Reply:
Right. I misunderstood the condition. As far as I can tell from Git and GitHub doc, `refs/heads/` is branches, so exactly what we want to query for. Keeping `startsWith...heads`, dropping the `!= true` part.